### DISCUSSION TITLE
Release v1.5.1 (beta 2)

### DISCUSSION THREAD

**Discussion by RevZero** [X2CommunityCore/xcom2-launcher#383] [-Releases-]
**Download:** [XCOM2_AlternativeModLauncher_1.5.1-beta.2.zip](https://github.com/X2CommunityCore/xcom2-launcher/releases/download/v1.5.1-beta.2/XCOM2_AlternativeModLauncher_1.5.1-beta.2.zip) 
> **Note that this is a beta-release and may contain bugs. Consider waiting for the final release to get a more polished experience.**

## Preface

Minor fix for previous beta release for mod change-logs not working again, because of recent Steam changes.

## Infos

Refer to the [installation instructions](https://github.com/X2CommunityCore/xcom2-launcher/wiki/Installation) about how to install or update AML.

- Check out [the Wiki](https://github.com/X2CommunityCore/xcom2-launcher/wiki) for some useful tips.
- For questions or suggestions you are welcome to join our [Discord channel](https://discord.gg/QHSVGRn) or start a [Discussion](https://github.com/X2CommunityCore/xcom2-launcher/discussions).
- If AML is crashing or otherwise not working as expected, you can [report a bug here](https://github.com/X2CommunityCore/xcom2-launcher/issues/new/choose).

## Changelog  

* Fixed mod change-logs not working because of Steam site layout changes. Fixes #374. @Alyssa4854 

<hr /><em>This discussion was created from the release <a href='https://github.com/X2CommunityCore/xcom2-launcher/releases/tag/v1.5.1-beta.2'>Release v1.5.1 (beta 2)</a>.</em>

**Comment by caseyjp11** [comment]
The link to the main zip is dead. "file not found"

**Reply by RevZero** [reply]
Thx, fixed!
